packages/transaction-controller/src/TransactionController.ts

```diff
@@ -35,6 +35,7 @@ import { addHexPrefix, bufferToHex } from 'ethereumjs-util';
 import { EventEmitter } from 'events';
 import { merge, pickBy } from 'lodash';
 import NonceTracker from 'nonce-tracker';
+import type { NonceLock } from 'nonce-tracker/dist/NonceTracker';
```

Comment:
Just for audit sake, from memory I think this is unavoidable as the package doesn't export the type in the index?

Is that correct?

Reply:
@matthewwalsh0 That is correct, unavoidable for now.

Reply:
@legobeat What about if I create a ticket to upgrade the dependency in a separate PR as we are at the end of reaching parity with the Transaction Controller in the extension, additionally the extension is using `v1.0.0`.